PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java

```diff
@@ -139,6 +144,7 @@ public void onBrowserSwitchResult(int requestCode, BrowserSwitchResult result, U
             try {
                 json.put("path", returnUri.getPath());
                 json.put("queryItems", queryItems);
+                json.put("fragment", fragment);
```

Comment:
looks like ios is calling this `hash`, right? https://github.com/braintree/popup-bridge-ios/commit/cd45320#diff-7d6a776faf444a83619c6174c700ef2bR138